### DISCUSSION TITLE
How do you intend to use Flock and Nest?

### DISCUSSION THREAD

**Discussion by matthew-carroll** [join-the-flock/nest#21] [Polls]
We're creating Flock, a community fork of Flutter, and Nest, tools for forking Flutter.

Flock is intended to be extremely easy to use. As simple as `fvm flutter run`. Flock will include various bug fixes and features that aren't available in Flutter.

Nest is a set of tools that make it easier to maintain a fork of Flutter. Maintaining a fork includes a number of challenges. These challenges will never be eliminated, but we believe that Nest will make them easier to deal with.

**Comment by matthew-carroll** [comment]
With about 50 votes, I'll summarize the findings.

We initially thought there might be a disproportionate number of users who intend to only use Nest, and not use Flock. The poll results show something closer to the opposite. On a spectrum, the results look somewhat like the following:

```
|   
|
|       |
|       |       |
|       |       |       |       |
Flock.        <--->          Nest
```

These numbers might change over time as people start using these things for real in production. However, what seems very clear, is that we can't skimp on support for Flock in favor of Nest. Both Flock and Nest will be used, and both will require significant attention towards ergonomics and capabilities.

**Comment by fisforfaheem** [comment]
Kindly start releasing new version and tell us what the team will fix [ and also add pull requests to original github[flutter[]
